### PR TITLE
Sam/documenation generator logic

### PR DESCRIPTION
**Pull request by SamanthaVS-BBD**
Logic for extracting info from file

### CODE REVIEW

**Review by jennadunfordbbd — APPROVED** [review]
nice

File: Api/DocumentGeneration.BFF.API.csproj [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	<PackageReference Include="Microsoft.Identity.Web.DownstreamApi" Version="2.15.2" />
+	<PackageReference Include="Serilog" Version="3.1.1" />
+	<PackageReference Include="Serilog.AspNetCore" Version="8.0.1" />
+	<PackageReference Include="Serilog.Enrichers.AssemblyName" Version="1.0.9" />
+	<PackageReference Include="Serilog.Enrichers.Environment" Version="2.3.0" />
+	<PackageReference Include="Serilog.Enrichers.Process" Version="2.0.2" />
+	<PackageReference Include="Serilog.Enrichers.Thread" Version="3.1.0" />
+	<PackageReference Include="Serilog.Extensions.Hosting" Version="8.0.0" />
+	<PackageReference Include="Serilog.Settings.Configuration" Version="8.0.0" />
+	<PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
```
Comment (seanvanwyk-bbd) [inline Api/DocumentGeneration.BFF.API.csproj:29]: 11 and 29 both have AspNetCore?
And the indentation is a bit funky. It runs though :)

File: DocumentGeneration.BFF.Core/Models/documentBaseClass.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace DocumentGeneration.BFF.Core.Models
+{
+    public class documentBaseClass
+    {
+        public string Name { get; set; }
+        public string Type { get; set; }
+        public string AccessModifier { get; set; }
+        
+        public List<documentFeildClass> Feilds { get; set; } = new List<documentFeildClass>();
```
Comment (seanvanwyk-bbd) [inline DocumentGeneration.BFF.Core/Models/documentBaseClass.cs]: Feild instead of Field. Other instances are probably also found, check with CTRL-F?

**Review by seanvanwyk-bbd — CHANGES_REQUESTED** [review]
Major setup seems good, just checking some house keeping

**Pull request merged**